packages/contracts/package.json

```diff
@@ -1,11 +1,22 @@
 {
-  "name": "@aragon/todo-osx-plugin-contracts",
-  "license": "AGPL-3.0-or-later",
-  "description": "",
+  "name": "@aragon/osx-plugin-template-contracts",
```

Comment:
should we name it `my-plugin` already? same for subgraph?

Reply:
or remind dev to change/rename this in the read me?

Reply:
No, I would say this is something for devs to take care about themselves.